README.md

```diff
@@ -75,6 +75,38 @@ The deployment's status can be followed through
   one, it also needs to be added to `templates/processbot.yaml`.
  - Secrets are managed through Gitlab.
 
+# Local development
```

Comment:
The README should take into account the master's README (it's outdated at the moment)